src/Symfony/Component/Console/Command/Command.php

```diff
@@ -426,6 +447,21 @@ public function addArgument(string $name, int $mode = null, string $description
         return $this;
     }
 
+    /**
+     * Set values for input completion.
+     *
+     * @throws InvalidArgumentException If the argument is not defined
+     *
+     * @return $this
+     */
+    public function setArgumentValues(string $name, \Closure|iterable|null $values): static
```

Comment:
what about adding an argument to `addArgument` instead of adding a method here?

Reply:
I chose to use a new method because the argument can be verbose (inlined array or closure) and I think the user code it leaner with a dedicated statement.

If you think it should be a new argument, it would be be added virtually for BC.

(That would be a good use-case for named arguments in PHP because the `Command::addOption` has already 4 optional arguments.)

Reply:
it's strange to have to 1. addArgument(), then 2. setArgumentValues(). It gives too much API surface to the feature to me.